README.md

```diff
@@ -58,15 +58,6 @@ wget -N https://raw.githubusercontent.com/pi-hole/PADD/master/padd.sh
 
 ## Running Pi-hole in a Docker Container
 If you're running Pi-hole in a Docker Container you can use `padd.sh` this way:
```

Comment:
```suggestion
If you're running Pi-hole in the official Docker Container, `padd.sh` is pre-installed and named `padd`. It can be used with the following command:
```

Something like this